src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -553,28 +845,183 @@ export const Deposit: React.FC<DepositProps> = ({
     [contextDispatch],
   )
   const handlePercentClick = useCallback(
-    (percent: number) => handleSendMax(percent === 1),
-    [handleSendMax],
+    async (percent: number) => {
+      if (!contextDispatch) return
+      contextDispatch({ type: ThorchainSaversDepositActionType.SET_LOADING, payload: true })
+      setIsSendMax(percent === 1)
+
+      const _percentageCryptoAmountPrecisionBeforeTxFees =
+        bnOrZero(cryptoAmountAvailable).times(percent)
+
+      const estimateFeesQueryEnabled = Boolean(
+        fromAddress && accountId && isUtxoChainId(asset.chainId),
+      )
+      // Fees before fees isn't a typo, it's an actual flow:
```

Comment:
Wondering whether or not we only want to go with the "fees after fees" flow for max deposits and rely on the first check only otherwise?

Reply:
Refactored the "fees after fees" logic with rounding 100% to 99% instead, avoiding the case of cheaper max sends because of a diff UTXO input/output set